content/2. Ondersteunende-informatie/3. Processen/3. Ontwerpproces/1. Opstellen ontwerp/Schematechnieken/1. Klassendiagram.md

```diff
@@ -0,0 +1,163 @@
+---
+title:
+  - Het klassendiagram
+tags:
+  - UML
+  - Klassendiagram
+difficulty: 2
+taxonomie:
+  - oo-15.1.Klassendiagram.OI
+  - oo-15.2.Klassendiagram.OI
+draft: true
+---
+
+## Wat is een klassendiagram
+
+Het klassendiagram is een ontwerptechniek voor het modelleren van de zogenaamde `structuur` van een softwaresysteem. Het bevat informatie over de `eigenschappen`(attributen) en het `gedrag` (methoden) van de klassen in het systeem. Ook bevat het diagram informatie over de `samenhang` (relaties) tussen de diverse klassen. 
+
+Onderstaand diagram is een voorbeeld van een klassendiagram. Hierin zie je de structuur van een eenvoudig hotel softwaresysteem. Dit voorbeeld is netjes uitwerkt maar kan zijn ontstaan terwijl je met je collega's bij een whiteboard stond.
+
+```mermaid
+classDiagram
+
+class Hotel {
+    +String naam
+    +String locatie
+    +geefKamers(bedrag)
+}
+
+
+class Kamer {
+	+String kamerNummer
+	+int aantalBedden
+	+decimal prijsPerNacht
+}
+
+Hotel --> Kamer
+```
+*Diagram 1: een eenvoudig voorbeeld van een klassendiagram*
```

Comment:
Hier moet de nummering van het diagram 1.1 zijn, want het zit in bestand nummer 1 en is het eerste diagram
```suggestion
*Diagram 1.1: een eenvoudig voorbeeld van een klassendiagram*
```